src/tests/eosjs-jssig.test.ts

```diff
@@ -9,17 +8,22 @@ describe('JsSignatureProvider', () => {
         'PUB_K1_5imfbmmHC83VRxLRTcvovviAc6LPpyszcDuKtkwka9e9Jg37Hp',
     ];
 
+    // These didn't test the correctness of signing. They also depend on the now-removed eosjs-ecc.
+
     it('builds public keys from private when constructed', async () => {
+        /*
```

Comment:
Rather than commenting out, I'd suggest just going ahead and deleting them.

Semi-related, if these tests didn't actually test the correctness it would be nice to see tests that do test the correctness.